contrib/building.md

```diff
@@ -51,6 +51,12 @@ The following dependencies are required. All except Xcode can be installed by fo
 - [Android NDK 21.0](https://developer.android.com/ndk/downloads/index.html)
 - [Android CMake](https://developer.android.com/ndk/guides/cmake)
 
+Moreover, in order to avoid introducting false positives in our analytics dataset, it is highly recommended to disable analytics by add the following to your shell configuration:
```

Comment:
```suggestion
Moreover, in order to avoid introducing false positives in our analytics dataset, it is highly recommended to disable analytics by adding the following to your shell configuration:
```